config/config.hjson

```diff
@@ -90,4 +90,8 @@
 #    # whether or not smtp connections should use tls
 #    use_tls: true
 #  }
+  # additional_slurs:
+  #  '''
+  #  (\bThis\b)|(\bis\b)|(\bsample\b)
+  #  '''
```

Comment:
The heredoc makes sense here *I think*, although hjson might be able to handle everything after the `:` fine.